types/errors.go

```diff
@@ -268,3 +281,18 @@ func (err *sdkError) QueryResult() abci.ResponseQuery {
 		Log:  err.ABCILog(),
 	}
 }
+
+func parseCmnError(err string) string {
+	if idx := strings.Index(err, "{"); idx != -1 {
+		err = err[idx+1 : len(err)-1]
+	}
```

Comment:
This isn't a good way of doing this. This means when someone comes around to update the sdk to the next tm version, they won't see a break here, which means were more likely to have this legacy code laying around forever. It should have been something that will break, with a comment for how to upgrade imo.

@cwgoes when we have multiple people reviewing can we wait a bit between last update and merge? (Timezones meant I didn't see it between last update and merge)

Reply:
Generally I agree - @fedekunze didn't want to wait for the Tendermint PR.

Reply:
https://github.com/cosmos/cosmos-sdk/issues/2133

Reply:
I agree with merging this with out waiting for tendermint. However we want to know to delete this code in the next release, not have it lay around forever. (Which is practically realized by a compilation fault or a test failure) a comment would have sufficed with the old method, this method risks legacy code remaining indefinitely. (It's not a big deal, just would have preferred it getting caught before this was merged)